sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1118,17 +1118,13 @@ public partial class ServerCertificateCustomValidationArgs
 }
 namespace Azure.Core.Serialization
 {
-    public partial interface IJsonSerializable
-    {
-        void Deserialize(System.IO.Stream stream, Azure.Core.Serialization.SerializableOptions? options = null);
-        void Serialize(System.IO.Stream stream, Azure.Core.Serialization.SerializableOptions? options = null);
-        bool TryDeserialize(System.IO.Stream stream, out long bytesConsumed, Azure.Core.Serialization.SerializableOptions? options = null);
-        bool TrySerialize(System.IO.Stream stream, out long bytesWritten, Azure.Core.Serialization.SerializableOptions? options = null);
-    }
     public partial interface IMemberNameConverter
     {
         string? ConvertMemberName(System.Reflection.MemberInfo member);
     }
+    public partial interface IModelSerializable
```

Comment:
Are we concerned about having an interface with no members?

There is a guideline that says:
> ❌ AVOID using marker interfaces (interfaces with no members).
>
>If you need to mark a class as having a specific characteristic (marker), in general, use a custom attribute rather than an interface.

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/interface

Reply:
base class would not be feasible since we need it across different class types such as ArmResource and Model classes.

We needed to go public with the internal methods for other reasons so closing this comment.